Makefile

```diff
@@ -248,10 +252,10 @@ $(DIST_DIR)/natcap.invest%.zip: | $(DIST_DIR)
 # on Windows as the .exe extension is assumed.
 binaries: $(INVEST_BINARIES_DIR)
 $(INVEST_BINARIES_DIR): | $(DIST_DIR) $(BUILD_DIR)
-	-$(RMDIR) $(BUILD_DIR)/pyi-build
-	-$(RMDIR) $(INVEST_BINARIES_DIR)
+	-rm -r $(BUILD_DIR)/pyi-build
```

Comment:
Can these use the `RMDIR` variable defined on lines 27/52?

Reply:
Fixed here: https://github.com/natcap/invest/pull/314/commits/f8a92c70862d88d7dc5889e69dd4afa1141f4d87